game.h

```diff
@@ -59,6 +59,10 @@ I can see what the Item.cpp docs look like.
 class Game
 {
 	friend class Faction;
+	// Rather than making all of the functions & variables public, we can allow this special class to access them.
+	// This class won't be defined except in the unit test code, though it could be used in other places if there
+	// were a good reason to do so.
+	friend class UnitTestHelper;
```

Comment:
this was the hack that I mentioned which allows me to do deeper unit testing where things need to get at internals of game but couldn't.  I honestly am a big believer in 'only make public the things that should be public', so will probably over time be privatizing things in other classes/structs that are only used internally, and will likely grant other UnitTestHelper friend status to those other classes if needed.